docs/guides/nwaku/upgrade-instructions.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Upgrade Instructions
+hide_table_of_contents: true
+sidebar_label: Upgrade Instructions
+---
+
+import { AccordionItem } from '@site/src/components/mdx'
+
+If you are currently using Nwaku, are running an old version and want to upgrade your node, please follow the below migration instructions for each release newer than your current running version
+
+<AccordionItem title="v0.31.0">
+Named sharding was deprecated in this version. This means that pubsub topics will only be supported if they comply with the static sharding format: <code>/waku/2/rs/&lt;CLUSTER_ID&gt;/&lt;SHARD_ID&gt;</code><br /><br />
+
+In order to migrate your existing application, you need to:
+
+1. Make sure that your clients are sending messages to pubsub topics in the required format.
+2. When running a node with the `--pubsub-topic` CLI flag, the values provided should comply with the static sharding format.
```

Comment:
Interesting if we could add an example about the "static sharding format".

Reply:
I added the definition of the static sharding format right above it, in https://github.com/waku-org/docs.waku.org/blob/8c289fb55e1f5037c233209c145a6cdebdad11fe/docs/guides/nwaku/upgrade-instructions.md?plain=1#L12

do you think we should repeat that point?